google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -1185,6 +1192,11 @@ private boolean isSessionNotFound(SpannerException e) {
     return e.getErrorCode() == ErrorCode.NOT_FOUND && e.getMessage().contains("Session not found");
   }
 
+  private boolean isDatabaseDoesNotExist(SpannerException e) {
```

Comment:
Can the logic be flipped and we call this isDatabaseExists() or databaseExists()? The suggestion is mostly for making sense of this function name.

Reply:
Or maybe isDatabaseNotFound() to match isSessionNotFound().

Reply:
Changed to `isDatabaseNotFound()` to keep in sync with `isSessionNotFound()`. Also changed the string we are checking for to "Database not found", as that is also the string we are checking for in `isSessionNotFound()` (and the text is included in the error message).